src/components/AddCar.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Navigate } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { addCar } from '../redux/carsReducer';
+
+export default function AddCar() {
+  const dispatch = useDispatch();
+  // const [value, setValue] = useState();
+
+  // const minYear = 2015;
+  // const maxYear = new Date().getFullYear();
+
+  // const handleChange = (event) => {
+  //   const value = Math.max(minYear, Math.min(maxYear, Number(event.target.value)));
+  //   setValue(value);
+  // };
```

Comment:
- [ ] Please, kindly remove this old piece of commented code in order to improve the cleanliness of the repo 💪